README.markdown

```diff
@@ -137,65 +151,67 @@ This is an alternative approach to using INSTSRV.EXE [1](http://support.microsof
 
 #### `registry::value`:
 
+All parameters are required unless otherwise stated.
+
 ##### `key`
 
-*Required.* Specifies a Registry key for Puppet to manage. Note that if any of the parent keys in the path do not exist, the resource raises an error. Use the `registry_key` to create the parent key prior to setting a registry value. Valid options: a string containing a Registry path.
+Specifies a Registry key for Puppet to manage. Note that if any of the parent keys in the path do not exist, the resource raises an error. Use the `registry_key` to create the parent key prior to setting a registry value. Valid options: a string containing a Registry path.
 
 ##### `data`
 
-*Required.* Provides the contents of the specified value. Valid options: a string by default; an array if specified through the `type` parameter.
+Provides the contents of the specified value. Valid options: a string by default; an array if specified through the `type` parameter.
 
 ##### `type`
 
-*Optional.* Sets the data type of the specified value. Valid options: 'string', 'array', 'dword', 'qword', 'binary', and 'expand'. Default value: 'string'.
+*Optional.* Sets the data type of the specified value. Valid options: `string`, `array`, `dword`, `qword`, `binary` and `expand`. Default value: `string`.
 
 ##### `value`
 
-*Optional.* Determines what Registry value(s) to manage within the specified key. To set a Registry value as the default value for its parent key, name the value '(default)'. Valid options: a string. Default value: the title of your declared resource.
+*Optional.* Determines what Registry value(s) to manage within the specified key. To set a Registry value as the default value for its parent key, name the value `(default)`. Valid options: a string. Default value: the title of your declared resource.
 
 #### `registry_key`
 
 ##### `ensure`
 
-Tells Puppet whether the key should or shouldn't exist. Valid options: 'present' and 'absent'. Default value: 'present'.
+Tells Puppet whether the key should or shouldn't exist. Valid options: `present` and `absent`. Default value: `present`.
```

Comment:
present and absent, as far as I know, are definitely are strings and should be in single quotes